docs/hydra/configure-deploy.mdx

```diff
@@ -61,7 +61,12 @@ We highly recommend using Docker to run Hydra, as installing, configuring and
 running Hydra is easiest with Docker. Ory Hydra is available on
 [Docker Hub](https://hub.docker.com/r/oryd/hydra/).
 
-```sh
+:::note
+This example remaps standard hydra ports `4444` and `4445` to prevent conflicts with a locally running hydra server.
```

Comment:
```suggestion
This example remaps standard hydra ports `4444` to `5444` and `4445` to `5445` to prevent conflicts with a locally running hydra server.
```